src/components/board/Board.vue

```diff
@@ -124,13 +126,51 @@ export default {
 		},
 	},
 	watch: {
-		id: 'fetchData',
+		id(newValue, oldValue) {
+			if (oldValue) {
+				// close old session
+				sessionApi.closeSession(oldValue, this.token)
+				this.token = null
+			}
+			// create new session
+			this.ensureSession(newValue)
+
+			this.fetchData()
+		},
 		showArchived() {
 			this.fetchData()
 		},
 	},
 	created() {
+		if (isNotifyPushEnabled()) {
+			// create a session
+			this.ensureSession()
+		}
+
 		this.fetchData()
+
+		if (isNotifyPushEnabled()) {
+			// regularly let the server know that we are still here
+			this.sessionInterval = setInterval(() => {
+				this.ensureSession()
+			}, 25 * 1000)
```

Comment:
This is a main concern I have with the session pinging approach, as this will add quite a lot of additional load to the Nextcloud server, especially if we starting pinging the session every 25 seconds. I don't have a good alternative in mind, but maybe we can lower this and also make sure that we don't send requests if the browser tab is not in the foreground.

Reply:
since 8f698e8c015a7c98898a514c7f0fd31203079d7d the session closing is much more reliable, so the timeout and pinging is less important and can happen much less often.
with 434b297755a5fe2498b95ade65908a0c31d285c8 it is now set to 90 seconds